### ISSUE TITLE
ReferenceError: regeneratorRuntime is not defined

### ISSUE THREAD

**Issue by kevart** [leecade/koa-webpack-middleware#41]
Getting:

```
/[..]/node_modules/koa-webpack-middleware/lib/devMiddleware.js:18
    var _ref = _asyncToGenerator(regeneratorRuntime.mark(function _callee(ctx, next) {
                                 ^

ReferenceError: regeneratorRuntime is not defined
    at /[..]/node_modules/koa-webpack-middleware/lib/devMiddleware.js:18:34
    at Object.exports.default [as devMiddleware] (/[..]/node_modules/koa-webpack-middleware/lib/devMiddleware.js:44:4)
    at Object.<anonymous> (/[..]/server/app.js:20:28)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.runMain (module.js:605:10)
    at run (bootstrap_node.js:425:7)

```

**"koa-webpack-middleware": "^1.0.3",**

Using `require` as babel is not set up (yet) on the backend. Would that cause the issue?

```
const webpackMiddleware = require('koa-webpack-middleware');
app = app
    .use(webpackMiddleware.devMiddleware(cfg, options))
    .use(webpackMiddleware.hotMiddleware(cfg));
```

**Comment by vijayst** [comment]
I am having the same issue

**Comment by hilongjw** [comment]
+1

**Comment by leecade** [comment]
fixed #36
